runner/src/config.rs

```diff
@@ -45,6 +45,7 @@ pub struct Log {
 #[serde(deny_unknown_fields)]
 pub struct Debug {
     pub max_firmware_exits: Option<usize>,
+    pub debug_compile: Option<bool>,
```

Comment:
I would tend to call that one simply `debug`, it is often how it is called in other projects.